pkg/cmd/auth/status/status.go

```diff
@@ -134,6 +144,12 @@ func statusRun(opts *StatusOptions) error {
 				tokenDisplay = token
 			}
 			addMsg("%s Token: %s", cs.SuccessIcon(), tokenDisplay)
+
+			if scopesHeader != "" {
+				addMsg("%s Token Scopes: %s", cs.SuccessIcon(), scopesHeader)
+			} else {
+				addMsg("%s Token Scopes: None found", cs.Red("X"))
```

Comment:
Rendering this as an error in red color might be misleading for Integration tokens and PAT v2, which have a concept of "permissions" but not scopes, and thus do not render any `x-oauth-scopes` response header.

Maybe it's best to just not render information about "scopes" in that case and instead print the type of token used?

Reply:
@mislav thanks, I made the changes and updated the PR body to reflect if we don't know the token type and offer more information. If you think it's clearer, I can remove it entirely in those cases. 

Thinking about this, I wonder if 
- the current output is perhaps not correct with a server-to-server-token, as we can't know if the token has the scopes needed for cli operations
- we should modify the message when we don't know the user's login (see the output from a fine-grained token)
- we should modify some of the other output when using a fine-grained token, where we also show an error message

I feel like these may be out of scope from the original issue, but wanted to bring them up.